### PR TITLE
fix: improve vs build tools detection, closes #3113

### LINKED ISSUE #3113: [bug] cargo tauri info always shows "Webview2 & Visual Studio Build Tools" not installed
### Describe the bug

cargo tauri info always shows "Webview2 & Visual Studio Build Tools" not installed. 

But actually I have installed them already:
image

image

### Reproduction

_No response_

### Expected behavior

_No response_

### Platform and versions

```shell
Operating System - Windows, version 10.0.19043 X64
Webview2 - Not installed
Visual Studio Build Tools - Not installed

Node.js environment
  Node.js - 14.18.2
  @tauri-apps/cli - 1.0.0-beta.10
  @tauri-apps/api - 1.0.0-beta.8

Global packages
  npm - 6.14.15
  pnpm - Not installed
  yarn - 1.22.17

Rust environment
  rustc - 1.57.0
  cargo - 1.57.0

Rust environment
  rustup - 1.24.3
  rustc - 1.57.0
  cargo - 1.57.0
  toolchain - stable-x86_64-pc-windows-msvc (environment override by RUSTUP_TOOLCHAIN)

App directory structure
/build
/node_modules
/public
/src
/src-tauri

App
  tauri.rs - 1.0.0-beta.8
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - ../build
  devPath - http://localhost:3000/
  framework - React
```

### Stack trace

_No response_

### Additional context
1) I use the next tauri-cli: cargo install tauri-cli --git https://github.com/tauri-apps/tauri.git --branch next
2) And the installer will download Webview2 runtime every time, even if I have run the installer many times:
image

### PR DESCRIPTION
**Pull request by amrbashir**
### What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [X] No

### Checklist
- [X] When resolving issues, they are referenced in the PR's title (e.g `fix #___, #___`)
- [X] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [X] I have added a convincing reason for adding this feature, if necessary

### Other information
Output looks like this
```
Operating System - Windows, version 10.0.19044 X64
Webview2 - 96.0.1054.62
Visual Studio Build Tools:
   - Visual Studio Community 2022
   - Visual Studio Build Tools 2019

Node.js environment
  Node.js - 16.13.1
  @tauri-apps/cli - 1.0.0-beta.10
  @tauri-apps/api - Not installed

Global packages
  npm - 8.1.2
  pnpm - 6.23.6
  yarn - 1.22.5

Rust environment
  rustc - 1.57.0
  cargo - 1.57.0

Rust environment
  rustup - 1.24.3
  rustc - 1.57.0
  cargo - 1.57.0
  toolchain - stable-x86_64-pc-windows-msvc 

App directory structure
/.changes
/.github
/examples
/src
/webview-dist
/webview-src

App
  tauri.rs - no manifest (no lockfile)
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: http: https: 'unsafe-eval' 'unsafe-inline'
  distDir - ../public
  devPath - http://localhost:5000/
  bundler - Rollup
```

### CODE REVIEW

File: tooling/cli.rs/src/info.rs [hunk 1]
```diff
@@ -306,63 +244,66 @@ fn webview2_version() -> crate::Result<Option<String>> {
...
-      .args(&["-NoProfile", "-Command"])
-      .arg("Install-Module VSSetup -Scope CurrentUser")
-      .output()?;
-    output = run_vs_setup_instance();
-  }
-  let output = output?;
-  let versions = if output.status.success() {
-    let stdout = String::from_utf8_lossy(&output.stdout);
-    let mut versions = Vec::new();
+const VSWHERE: &[u8] = include_bytes!("../vswhere.exe");
```
Comment (lucasfernog) [inline tooling/cli.rs/src/info.rs:258]: how often do you think we'll update this binary? I think we won't need to update it ever but if we do I think we could move it to binary-releases (just like we did with rustup).

Comment (amrbashir) [reply tooling/cli.rs/src/info.rs:258]: maybe once every year or two, it is very unlikely to update this binary, it is also hosted in github releases here https://github.com/microsoft/vswhere/releases so no need to host it ourselves.

**Review by lucasfernog — APPROVED** [review]

**Comment by lucasfernog** [comment]
I think I'll wait for the audit fixes to be merged before merging any other PR that might introduce conflicts.

File: tooling/cli.rs/src/info.rs [hunk 2]
```diff
@@ -429,22 +429,27 @@ pub fn command(_options: Options) -> Result<()> {
...
+    if build_tools.is_empty() {
+      InfoBlock {
+        section: false,
+        key: "Visual Studio Build Tools - Not installed",
+        value: None,
+        suffix: None,
+      }
+      .display();
+    } else {
+    }
```
Comment (amrbashir) [inline tooling/cli.rs/src/info.rs]:
```suggestion
```

**Pull request merged**